index.js

```diff
@@ -9,7 +9,8 @@ function simplify (data) {
         return acc
       }, {})
     }
-    if (type === 'list') {
+    if (type === 'list' || type === 'byteArray' || type === 'intArray' || type === 'longArray') {
+      if (!Array.isArray(value.value)) return []
```

Comment:
if it's annotated as an array but it's not an array we return an empty array ?
why is this valid ?